whelk-core/src/main/groovy/whelk/component/ElasticSearch.groovy

```diff
@@ -78,6 +89,23 @@ class ElasticSearch {
         // FIXME: upgrade httpClient (and use RequestConfig)- https://issues.apache.org/jira/browse/HTTPCLIENT-1418
         // httpParams.setParameter(ClientPNames.CONN_MANAGER_TIMEOUT, new Long(TIMEOUT_MS));
 
+
```

Comment:
Probably better to use a scheduledExecutorService instead? https://stackoverflow.com/questions/409932/java-timer-vs-executorservice

Reply:
Perhaps. But looking over that list, none of the issues are relevant here.
The one-thread thing is actually a benefit here (although the code intentionally guards against multiple retries at once too). Slight deviations in timing don't matter to us and everything that could potentially throw is already guarded in try/catch blocks.

Also, ExecutorServices are complicated and messy (imo). :(

Reply:
It would nice to set name for the timer and make it not daemon.
for example `new Timer("ElasticIndexingRetrier", false)`

Reply:
Done!